vmdb/db/migrate/20150630025128_create_tenant.rb

```diff
@@ -0,0 +1,17 @@
+class CreateTenant < ActiveRecord::Migration
+  def change
+    create_table :tenants do |t|
+      t.string :domain
+      t.string :subdomain
+      t.string :name
+      t.string :vmdb_name
```

Comment:
It was `company_name`. I thought tenant.company_name seemed redundant.
Let me know if you come up with a better name, I'll try and fix too